lib/srv/ctx.go

```diff
@@ -751,7 +750,20 @@ func (c *ServerContext) CheckSFTPAllowed() error {
 	if err != nil {
 		return trace.Wrap(err)
 	}
-	if !canStart {
+	// canStart will be true for non-moderated sessions. If canStart is false, check to
+	// see if the request has been approved through a moderated session next.
+	if canStart {
+		return nil
+	}
+	if registry == nil {
+		return errCannotStartUnattendedSession
```

Comment:
Should these errors be wrapped so we get a trace to within this function ?

Reply:
I actually was thinking about that and forgot to do something about it... These should probably be normal errors constructed from `errors.New` and wrapped when returned, yeah.